build/canary.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+
+const pkgFile = path.resolve(__dirname, '..', 'package.json');
+const pkg = JSON.parse(fs.readFileSync(pkgFile));
+
+const { CIRCLE_SHA1 } = process.env;
+assert(CIRCLE_SHA1, 'CIRCLE_SHA1 enviornment variable not set');
+
+// Shorten the SHA
+const GIT_SHA = CIRCLE_SHA1.substr(0, 7);
```

Comment:
Only thing that would be nice, is to auto generate a readme file with a link back to that commit in develop.

And if like @WilcoFiers suggests if we are able to get the change log, add that to the readme as well, as otherwise the numerous packages on npm will soon go out of context and someone has to look through commit history to figure out what each version has...

Reply:
I don't know why we'd add something to a README. We don't do that for "regular" versions, and it's not something I've seen other packages doing. I understand the desire to update the CHANGELOG, but think that doing so makes the README update unnecessary.

This said, if we need to do this, I'm happy to make the change if it's decided that we need it.

Reply:
Ha ha, I will leave it with @WilcoFiers to decide if we need a README or not. 

We can always get this out and enhance as needed. Not a blocker in anyway.